### ISSUE TITLE
Documentation out of date

### ISSUE THREAD

**Issue by stephen-holinaty-sp** [desktop/desktop#15730]
### The problem

the guide at [desktop](https://github.com/desktop/desktop)/[docs](https://github.com/desktop/desktop/tree/development/docs)/[integrations](https://github.com/desktop/desktop/tree/development/docs/integrations)/gitlab.md 

gives incorrect details on how to generate a PAT.

they are now under Settings > Developer Options > Personal Access Tokens

### Release version

current documentation

### Operating system

all (web doc)

### Steps to reproduce the behavior

open [desktop](https://github.com/desktop/desktop)/[docs](https://github.com/desktop/desktop/tree/development/docs)/[integrations](https://github.com/desktop/desktop/tree/development/docs/integrations)/gitlab.md

try to follow the guide

fail

get mad

search google

find real instructions

### Log files

n/a

### Screenshots

n/a

### Additional context

simple a documentation update request

**Comment by steveward** [comment]
Thanks for the report @stephen-holinaty-sp, I'll get this updated!

**Comment by steveward** [comment]
@stephen-holinaty-sp I took a look and it looks like not much has changed other than the profile dropdown changing from `Settings` to `Edit Profile`. I referenced [GitLab's documentation](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html)  to confirm their instructions as well. I've updated our doc to the `Edit Profile` option along with a new screenshot. 

Where are you seeing `Settings` > `Developer Options`?
